pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -21,11 +22,12 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against nodes.
-func NewStorage(s storage.Interface) *REST {
+func NewStorage(optsGetter restoptions.Getter) *REST {
```

Comment:
the rebase is changing this for every `NewStorage` to use `generic.RESTOption` to sync it with upstream, I wonder how the restoptions.Getter is different.